### DISCUSSION TITLE
Target class [Backpack\CRUD\app\Http\Middleware\AuthenticateSession] does not exist.

### DISCUSSION THREAD

**Discussion by tudoavinfo** [Laravel-Backpack/community-forum#342] [Q&A (Help)] [answered]
Target class [Backpack\CRUD\app\Http\Middleware\AuthenticateSession] does not exist.

**Comment by tabacitu** [comment]
Normally I'd say this is too little information for us to know what's going on. That we need to know what you did before, to get this error.

But oddly enough, this sounds familiar and I think @pxpm might know exactly what's going on.

Until he answers, please try a "composer update" and let us know more information if you can.

Cheers!

**Comment by pxpm** [comment]
This was fixed in https://github.com/Laravel-Backpack/CRUD/pull/4849 that was released yesterday in https://github.com/Laravel-Backpack/CRUD/releases/tag/5.4.12

Please make sure you are using that version. You can check what version you are using by doing: `composer show backpack/crud` or `php artisan backpack:version`.

Cheers

**Reply by cl3rO3Y** [reply]
Hello!
I'm not the person who wrote the first message, but I've got the same issue, the issue came once I've ran a 'composer update'. 
I've got the version that should however fix the problem: 
### BACKPACK VERSION:
5.4.12@6ff47ed84735c672cf08c763c633419beea8e392

Any help appreciated.

**Reply by pxpm** [reply]
The version that should fix it is not fixing it. 🙃 

Sorry about the bad experience, we have already the fix in place, @tabacitu should release it in the next hour orso. 

After, a composer update should definitely fix it for you 🙏 🤞

**Reply by tabacitu** [reply]
Merged & tagged. Please do a `composer update` to get 5.4.13 - that has it fixed!

**Reply by cl3rO3Y** [reply]
I can confirm it's perfectly working now! Thanks for the quick update!

**Comment by tabacitu** [comment, marked as answer]
Please do a `composer update backpack/crud` to get 5.4.13 or later - that has it fixed!
